packages/react/src/lib/AutoControlledComponent.tsx

```diff
@@ -169,14 +178,19 @@ export default class AutoControlledComponent<P = {}, S = {}> extends UIComponent
       return acc
     }, {})
 
-    this.state = { ...state, ...initialAutoControlledState, autoControlledProps }
+    this.state = {
+      ...state,
+      ...initialAutoControlledState,
+      autoControlledProps,
+      getAutoControlledStateFromProps,
```

Comment:
Isn't this a method? Should we invoke it to get the object containing props?

Reply:
https://github.com/stardust-ui/react/pull/1449#discussion_r290287824

It's an initial state 🤔 We store it in state and will be able to access it inside of `static`.

Reply:
Let's describe from user perspective how these methods should be defined and used in the PR description. I am a bit confused as why we are introducing this method, and how it should be used.

Reply:
Update PR description